src/server.js

```diff
@@ -314,19 +314,26 @@ const PATH_NAME = process.env.NODE_ENV === "production" ?
   '~/gazelle-production/scripts/restartServers.sh' : process.env.NODE_ENV === "beta" ?
   '~/gazelle-beta/scripts/restartServers.sh' : __dirname + '/scripts/restartServers.sh';
 
+var isrestarted = false;
+
 editorTools.get('/restartserver', (req, res) => {
+
+  isrestarted = true;
```

Comment:
You shouldn't set the variable before you check if the password is correct. Only do it when you know you're actually gonna restart the server.